server/base/src/main/java/org/apache/accumulo/server/metrics/Metrics.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.accumulo.server.metrics;
 
+import java.util.ServiceLoader;
+
+import org.apache.accumulo.server.metrics.service.MetricsRegistrationService;
 import org.apache.hadoop.metrics2.MetricsCollector;
```

Comment:
I'm of the opinion that the Hadoop Metrics2 stuff should be ripped out. If I'm understanding this correctly, we are publishing metrics via Micrometer and Hadoop Metrics concurrently. If both are configured to send metrics to the same destination, then we will be double reporting the numbers. IMO, Hadoop can send it's metrics via mechanism M and Micrometer can send it's metrics via mechanism M if the user so chooses.